mqtt_base_block.py

```diff
@@ -30,9 +30,8 @@ def stop(self):
 
     def connect(self):
         self.logger.debug("Connecting...")
-        self.client.connect(self.host(), self.port())
         self.client.on_connect = self.on_connect
-        # ^ will this work if on_connect is only defined in the sub/pub blocks?
```

Comment:
`loop_stop` is inside `disconnect` so shouldn't `loop_start` be inside `connect`? or vice versa.